confidant/ciphermanager.py

```diff
@@ -22,7 +25,7 @@ def encrypt(self, raw):
             log.warning('Not using encryption in CipherManager.encrypt'
                         ' If you are not running in a development or test'
                         ' environment, this should not be happening!')
-            return raw
+            return 'UNENCRYPTED_{0}'.format(base64.b64encode(raw))
```

Comment:
What would you think about making the unencrypted prefix a constant instead of hardcoding it twice here?

Also, thoughts on something more obvious? `DANGER_NOT_ENCRYPTED_{0}`? :)